README-template.md

```diff
@@ -62,13 +66,26 @@ This project is maintained by [Greenbone Networks GmbH](https://www.greenbone.ne
 
 ## Contributing
 
-Your contributions are highly appreciated. Please [create a pull request](https://github.com/greenbone/X/pulls) on GitHub. For bigger changes, please discuss it first in the [issues](https://github.com/greenbone/X/issues).
+Your contributions are highly appreciated. Please [create a pull request](https://github.com/greenbone/X/pulls) on GitHub. For bigger changes, please start a discussion with the development team via the [issues section at github](https://github.com/greenbone/X/issues) first.
 
 TODO: Replace X with project references!
 
 State here if contributions are welcome. State the requirements a contribution should meet to get merged.
 
-Details about development, for example creating a dev environment or running tests, also belongs here.
+Details about development, like creating a dev environment or running tests, also belong here, for example:
+
+For development you should use [pipenv](https://pipenv.readthedocs.io/en/latest/) to keep you python packages separated in different environments. First install pipenv via pip
+
+    pip install --user pipenv
+
+Afterwards run
+
+    pipenv install --dev
+
+in the checkout directory of X (the directory containing the Pipfile) to install all dependencies including the packages only required for development.
+
+If there are more specific suggestions for development or suggestions only relevant to specific environments or use cases, consider creating a separate DEVELOPMENT.md file and [linking it here](DEVELOPMENT.md).
```

Comment:
A common file for that is CONTRIBUTING.md. It is even displayed to new contributors on GitHub when creating a PR.

https://help.github.com/articles/setting-guidelines-for-repository-contributors/

Reply:
I was thinking more along the lines of providing suggestions for the development process than for contributing the results of this process.

But I think this is too specific for a template, I rephrased the sentence to be more generic and less focused on a particular file or format.